### ISSUE TITLE
ValueError: Namespace GtkSource not available for version 5

### ISSUE THREAD

**Issue by glher** [gaphor/gaphor#2689]
I installed Gaphor using "pip install gaphor" (python3.10).

However, when trying to open it ($ gaphor in terminal), I get:

```
Traceback (most recent call last):
  File "/home/user/Projects/.virtualenvs/gaphor/bin/gaphor", line 8, in <module>
    sys.exit(main())
  File "/home/user/Projects/.virtualenvs/gaphor/lib/python3.10/site-packages/gaphor/main.py", line 21, in main
    commands: dict[str, argparse.ArgumentParser] = initialize("gaphor.argparsers")
  File "/home/user/Projects/.virtualenvs/gaphor/lib/python3.10/site-packages/gaphor/entrypoint.py", line 13, in initialize
    return init_entry_points(load_entry_points(scope, services), **known_services)
  File "/home/user/Projects/.virtualenvs/gaphor/lib/python3.10/site-packages/gaphor/entrypoint.py", line 29, in load_entry_points
    cls = ep.load()
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 171, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/user/Projects/.virtualenvs/gaphor/lib/python3.10/site-packages/gaphor/plugins/diagramexport/__init__.py", line 15, in <module>
    from gaphor.ui.filedialog import save_file_dialog
  File "/home/user/Projects/.virtualenvs/gaphor/lib/python3.10/site-packages/gaphor/ui/__init__.py", line 11, in <module>
    gi.require_version("GtkSource", "5")
  File "/home/user/Projects/.virtualenvs/gaphor/lib/python3.10/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available for version %s' %
ValueError: Namespace GtkSource not available for version 5
```

I indeed do not seem to have GTK5 installed. However, I thought from the doc that v4 was enough (I may have misunderstood). When I go to a python console, I can get:

```
>>> import gi
>>> gi.require_version('Gtk', '4.0')
>>> from gi.repository import Gtk
>>> Gtk.MAJOR_VERSION
4
```

and

```
>>> import gi
>>> gi.require_version('Gtk', '5.0')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/glher/Projects/.virtualenvs/gaphor/lib/python3.10/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available for version %s' %
ValueError: Namespace Gtk not available for version 5.0

```

I have a Pop OS 22.04 (jammy). I have been unable to track down the correct sudo apt install for GtkSource version 5. Would you be able to point me in the right direction?

**Comment by glher** [comment]
Note: Installing through flatpack works like a breeze

**Comment by amolenaar** [comment]
Hi @glher,

Thank you for reaching out.

The required GTK version is 4.0. For  [GtkSourceView](https://gitlab.gnome.org/GNOME/gtksourceview) we require version 5.x.

The apt package name is ` libgtksourceview-5-0`.

**Comment by danyeaw** [comment]
Hi @glher I didn't see a response from you, and the dependencies are listed here:
https://docs.gaphor.org/en/latest/linux.html

Please feel free to reopen the issue if you installed all the dependencies, and you are still having issues, thanks!

**Comment by glher** [comment]
Yes, sorry I did not get back to you sooner! This was indeed a misread on my part, between GTK itself and GTtkSourceView in terms of version number. I have installed libgtksourceview-5-0 (sudo apt-get install libgtksourceview-5-0). It did not seem to solve the issue, but that's more likely a user error at this point. I have not spent more time exploring this, as installing via the flatpack went super smoothly and everything works. Thanks for your great work!
